include/seqan3/io/alignment_file/all.hpp

```diff
@@ -0,0 +1,29 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief [DEPRECATED] Meta-include for the alignment IO submodule.
+ * \deprecated This header will be removed in 3.1.0; Please \#include <seqan3/io/sam_file/all.hpp> instead.
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/io/sam_file/all.hpp>
+
+#include <seqan3/io/alignment_file/format_bam.hpp>
+#include <seqan3/io/alignment_file/format_sam.hpp>
+#include <seqan3/io/alignment_file/header.hpp>
+#include <seqan3/io/alignment_file/input.hpp>
+#include <seqan3/io/alignment_file/input_format_concept.hpp>
+#include <seqan3/io/alignment_file/input_options.hpp>
+#include <seqan3/io/alignment_file/output.hpp>
+#include <seqan3/io/alignment_file/output_format_concept.hpp>
+#include <seqan3/io/alignment_file/output_options.hpp>
+#include <seqan3/io/alignment_file/sam_tag_dictionary.hpp>
```

Comment:
Are these includes still needed after ```#include <seqan3/io/sam_file/all.hpp>```?

Reply:
`all.hpp` won't completely work, because some headers like `#include <seqan3/io/alignment_file/input.hpp>` define the deprecated `seqan3::alignment_file_input`.

So to be safe I just used the old include list without changing anything.

Reply:
yeah, xD